addons/common/functions/fnc_showHud.sqf

```diff
@@ -0,0 +1,58 @@
+/*
+ * Author: PabstMirror
+ * Allows multiple sources to not overwrite showHud command
+ * Bitwise AND Logic (a single false in a mask will make it false)
+ *
+ * Arguments:
+ * 0: Source ID <STRING><OPTIONAL>
+ * 1: Show Hud Bool Array (8 to set, empty to remove) <ARRAY><OPTIONAL>
```

Comment:
I think this could do with some details about this array. What do those 8 values mean?